### PR TITLE
fix typo -> {{}} needs to be prefixed by $

### PR DESCRIPTION
**Pull request by fkwp**

### CODE REVIEW

File: .github/workflows/publish.yaml [hunk 1]
```diff
@@ -69,7 +69,7 @@ jobs:
             type=sha,format=short,event=branch
             type=semver,pattern={{version}}
             type=raw,value=latest-ci,enable={{is_default_branch}}
-            type=raw,value=latest-ci_{{steps.current-time.outputs.unix_time}},enable={{is_default_branch}}
+            type=raw,value=latest-ci_${{steps.current-time.outputs.unix_time}},enable={{is_default_branch}}
```
Comment (robintown) [inline .github/workflows/publish.yaml:72]: Does that fix need to apply to all the other uses of `{{}}` in these lines, too?

Comment (fkwp) [reply .github/workflows/publish.yaml:72]: tbh. I don't have an idea... I saw lines with and without $

Comment (fkwp) [reply .github/workflows/publish.yaml:72]: https://docs.github.com/en/actions/learn-github-actions/expressions

Comment (robintown) [reply .github/workflows/publish.yaml:72]: Okay, well let's try this and see what it does

**Review by robintown — APPROVED** [review]

**Pull request merged**